lib/ClaimVerifier.ts

```diff
@@ -32,6 +33,7 @@ export class ClaimVerifier {
     const filteredRoles = roles.filter(Boolean) as {
       name: IRoleDefinitionV2['roleName'];
       namespace: RoleEIP191JWT['payload']['claimData']['claimType'];
+      error: string;
```

Comment:
Its still looks odd to me that `getVerifiedRoles` returns not verified role

Reply:
1. To me verification is about ensuring the data came from the right channel and its integrity is still persisted. Validity is solely based on business use case and it is next step after verification. For example - I was issued driving license which was valid for 10 years. Even after expiration I can use it to prove my age at a Bar but I cannot drive without renewal.
2. `getVerifiedRoles` is solely responsible for verification and if one wants to also know the invalid roles (expired or revoked) then it will come from this call itself. 

Maybe we can segregate the valid and revoked / expired role here itself so that there is no confusion.